js/ui/speed_dial_action/speed_dial_action.js

```diff
@@ -30,6 +30,13 @@ const SpeedDialAction = Widget.inherit({
 
             onClick: null,
 
+            /**
+            * @name dxSpeedDialActionOptions.label
+            * @type string
+            * @default ""
+            */
+            label: "",
```

Comment:
Fixed. Position changes according to FAB position if the position set is left label shows on the right.

Reply:
We have added some magic instead of adding a simple option. 'left' in position.at does not mean that I want to render the labels to the right. Anyway, the 'at' option can contain 'x' and 'y' (https://js.devexpress.com/Documentation/ApiReference/Common/Object_Structures/positionConfig/at/) and you code [here](https://github.com/DevExpress/DevExtreme/pull/9093/files#diff-a57151fa32bbc9ff906be3840d2c859bR70) can cause the error when you try to call `startsWith` on the object.